README.md

```diff
@@ -13,9 +13,9 @@ Update status and "latest release" badges:
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 <!-- Add a description of module(s) in this repo -->
-Use this module to install a WebSphere® Liberty operator and create an instance of WebSphere Liberty operator on IBM Cloud Red Hat OpenShift cluster on VPC Gen2.
+Use this module to install WebSphere Application Server Liberty on IBM Cloud Red Hat OpenShift Container Platform on VPC landing zone by deploying IBM WebSphere Liberty operator.
```

Comment:
Perhaps, `Use this module to install a WebSphere Application Server Liberty on the IBM Cloud Red Hat OpenShift Cluster`?

Reply:
what about `Use this module to install WebSphere Application Server Liberty in IBM Cloud Red Hat OpenShift Container Platform on VPC landing zone: this module deploys and configures the IBM WebSphere Liberty operator into the IBM Cloud Red Hat OpenShift Cluster.`
?

@SirSpidey needing your help on this :-)

Reply:
I think either of these 
> Use this module to install a WebSphere Liberty operator on your Red Hat OpenShift Container Platform on VPC landing zone.

> Use this module to install WebSphere Application Server Liberty on your Red Hat OpenShift Container Platform on VPC landing zone.

assuming that it is installing on an existing DA.

Reply:
@SirSpidey thank you, adopted the first one